.github/workflows/self-past.yml

```diff
@@ -126,6 +126,12 @@ jobs:
         run: |
           nvidia-smi
 
+      - name: Install
+        if: inputs.framework == 'pytorch'
+        working-directory: /transformers
+        run: |
+          python3 -m pip install --no-cache-dir git+https://github.com/huggingface/accelerate@main#egg=accelerate
```

Comment:
need this to be installed in CI jobs, not in Past CI docker images. See PR description.

Reply:
Ok ty

Reply:
Appreciate it